### ISSUE TITLE
Reading passed parameters in JS file

### ISSUE THREAD

**Issue by gyosida** [mobile-dev-inc/maestro-docs#55]
The documentation indicates that is possible [to pass parameters to JS file using runScript as with runFlow](https://github.com/mobile-dev-inc/maestro-docs/blob/main/advanced/javascript/run-javascript.md#passing-parameters). Provides an example and that’s good.

However, having an example to read the parameters in the JS file would be handy. Had to google and find a random repository to know that I had to directly reference the name of the parameter and not ${myParameter}, or process.env.myParameter.

Passing parameters

```
- runScript:
     file: script.js
     env:
        myParameter: ‘Parameter’
```

Reading a parameter in a JS file
```
const passedInParameter = myParameter;
```

**Comment by Fishbowler** [comment]
Fixed by #62

**Comment by bartekpacia** [comment]
thanks Dan!
